### PR TITLE
Schema for pipe transport options.

### PR DESCRIPTION
**Pull request by rajkumar42**
Schema for pipe transport options.

### CODE REVIEW

**Comment by rajkumar42** [comment]
@chuckries @gregg-miskelly @caslan @DustinCampbell

**Comment by gregg-miskelly** [comment]
Might be obvious, but just in case - I would recommend waiting till you have an official build of OpenDebugAD7 with your changes that you can change the following line to pull in your changes:

    src\coreclr-debug\main.ts:337:            "Microsoft.VisualStudio.OpenDebugAD7": "1.0.20520-preview-3"

File: package.json [hunk 1]
```diff
@@ -280,6 +280,86 @@
                   "type": "string"
                 },
                 "default": []
+              },
+              "pipeTransport": {
+                "type": "object",
+                "description": "Describes options to launch a program through a pipe transport.",
```
Comment (gregg-miskelly) [inline package.json]: How about:When present, this tells the debugger to connect to a remote computer using another executable as a pipe that will relay standard input/output between VS Code and the .NET Core debugger backend executable (clrdbg).

File: package.json [hunk 2]
```diff
@@ -280,6 +280,86 @@
...
+                "properties" : {
+                  "pipeProgram": {
+                    "type": "string",
+                    "description": "The pipe command to execute.",
+                    "default": "enter pipe program name, for example plink.exe"
+                  },
+                  "pipeArgs": {
+                    "type": "array",
+                    "description": "Command line arguments passed to the pipe program.",
+                    "default": []
```
Comment (gregg-miskelly) [inline package.json]: Are you missing?

                  "items": { 
                    "type": "string" 
                  },

File: package.json [hunk 3]
```diff
@@ -280,6 +280,86 @@
...
+              },
+              "pipeTransport": {
+                "type": "object",
+                "description": "Describes options to launch a program through a pipe transport.",
+                "default": {
+                  "pipeProgram": "enter pipe program name, for example plink.exe",
+                  "pipeArgs": []
+                },
+                "properties" : {
+                  "pipeProgram": {
```
Comment (gregg-miskelly) [inline package.json]: Does the MIEngine search PATH if this is not a full file path? Or does it need to be a file path? Probably worth calling out in the description either way.

**Comment by gregg-miskelly** [comment]
Otherwise looks good to me.

File: package.json (hunk 1 above)
Comment (rajkumar42) [inline package.json]: done.

File: package.json (hunk 2 above)
Comment (rajkumar42) [inline package.json]: fixed.

File: package.json (hunk 3 above)
Comment (rajkumar42) [inline package.json]: fixed.

**Pull request merged**